src/components/BalanceToken.js

```diff
@@ -47,7 +47,7 @@ const BalanceToken = ({ amount, color, decimals, icon, size, symbol }) => {
   )
 }
 
-const TokenIcon = styled.img.attrs({ alt: '', width: '20', height: '20' })`
+const TokenIcon = styled.img.attrs({ alt: '', width: '24', height: '24' })`
```

Comment:
are we using this image like this for some reason?

Reply:
Think this was forked from the original Aragon client Conviction voting app and didn't see the reason to change it.